packages/arb-token-bridge-ui/src/pages/api/denylist.ts

```diff
@@ -0,0 +1,44 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+import denylist from '../../../public/__auto-generated-denylist.json'
+
+const ONE_WEEK_IN_SECONDS = 60 * 60 * 24 * 7
+
+type Request = NextApiRequest & { query: { address: string } }
+
+export default async function handler(
+  req: Request,
+  res: NextApiResponse<{ data: boolean | undefined; message?: string }>
+) {
+  try {
+    // validate method
+    if (req.method !== 'GET') {
+      res
+        .status(400)
+        .send({ message: `invalid_method: ${req.method}`, data: undefined })
+      return
+    }
+
+    const { address } = req.query
+
+    if (typeof address !== 'string') {
+      res.status(500).send({
```

Comment:
should be 400, not 500, as it's a client error (malformed request)

https://developer.mozilla.org/en-US/docs/Web/HTTP/Status